releasenotes/notes/clib-definition-noname-75d0daf1355f4038.yaml

```
@@ -0,0 +1,7 @@
+---
+upgrade_circuits:
+  - |
+    The name of the circuit produced as definitions of circuit library gates were set to ``None``.
```

I am not sure which words in this sentence should be singular-vs-plural ("names"? "circuits"? "definitions"? "gates"?) but I do believe that it should be either "The name ... [was or is] set" or "The names ... [were or are] set".

```suggestion
    The names of the circuits produced as definitions of standard circuit library gates are now set to ``None``.
```